protocol/messenger_communities.go

```diff
@@ -659,6 +666,16 @@ func (m *Messenger) JoinCommunity(ctx context.Context, communityID types.HexByte
 	return mr, nil
 }
 
+func (m *Messenger) subscribeToCommunityShard(communityID []byte) error {
+	// TODO: store private key and topic
+	// TODO: determine pubsub topic and public key for community
+	// TODO: this should probably be moved completely to transport once pubsub topic logic is implemented
+
+	pubsubTopic := transport.GetPubsubTopic(communityID)
+	var communityPubKey *ecdsa.PublicKey
+	return m.transport.SubscribeToPubsubTopic(pubsubTopic, communityPubKey)
```

Comment:
I guess it's not implemented, but I would just pass `nil` here, so it looks a bit less like an error